contrib/init/dogecoind.opt.service

```
@@ -0,0 +1,74 @@
+# This variant of the unit file is for "opt" add-on installations that do not form part of the default installation.
+# (i.e. out of band installations by the user, not installed by a system package manager like "apt")
+#
+# The relevant paths are:
+#
+#/opt/dogecoin/dogecoind
+#/etc/opt/dogecoin/
+#/var/opt/dogecoin/
+
+[Unit]
+Description=Dogecoin's distributed currency daemon
+After=network.target
+
+[Service]
+Type=simple
+ExecStart=/opt/dogecoin/dogecoind -conf=/etc/opt/dogecoin/dogecoin.conf -datadir=/var/opt/dogecoin
```

```suggestion
ExecStart=/opt/dogecoin/bin/dogecoind -conf=/etc/opt/dogecoin/dogecoin.conf -datadir=/var/opt/dogecoin
```

`/opt/dogecoin` will keep the tarball structure intact, so the path to dogecoind is `/opt/dogecoin/bin/dogecoind`